spec/models/group_spec.rb

```diff
@@ -411,17 +411,37 @@ def real_staff
     expect(g.usernames.split(",").sort).to eq usernames.split(",").sort
   end
 
-  it "correctly destroys groups" do
+  describe 'new' do
+    subject { Fabricate.build(:group) }
 
-    g = Fabricate(:group)
-    u1 = Fabricate(:user)
-    g.add(u1)
-    g.save!
+    it 'triggers a extensibility event' do
+      event = DiscourseEvent.track_events { subject.save! }.first
+
+      expect(event[:event_name]).to eq(:group_created)
+      expect(event[:params].first).to eq(subject)
+    end
+  end
+
+  describe 'destroy' do
+    before do
+      @group = Fabricate(:group)
+      @user = Fabricate(:user)
+      @group.add(@user)
+      @group.save!
+    end
 
-    g.destroy
+    it "it deleted correctly" do
+      @group.destroy
```

Comment:
Generally it is preferred to use `destroy!` over `destroy` anytime you're not checking for the return value